platform/mbed_lib.json

```diff
@@ -108,11 +108,6 @@
             "value": null
         },
 
-        "error-decode-http-url-str": {
```

Comment:
I removed this config as its used only in one place. So I'm just hard-coding for now.

Reply:
That's fine, but because this is a config removal, this change would cause it to be bumped to 5.12.

Will assume that is the intention for now, unless this change is reverted to remain.